app/jinja_filters.py

```diff
@@ -1,41 +1,48 @@
 # coding: utf-8
 import re
+from decimal import Decimal
+from typing import Any, Mapping, Optional, Union
 
 import flask
 import flask_babel
 from babel import numbers, units
 from flask import current_app
 from jinja2 import pass_eval_context
 from markupsafe import Markup, escape
+from wtforms import SelectFieldBase
 
 from app.questionnaire.rules.utils import parse_datetime
 from app.settings import MAX_NUMBER
 
 blueprint = flask.Blueprint("filters", __name__)
 
 
-def mark_safe(context, value):
+def mark_safe(context: Any, value: str) -> str:
     return Markup(value) if context.autoescape else value
 
 
-def strip_tags(value):
+def strip_tags(value: str) -> str:
```

Comment:
This returns escaped Markup if you check escape() method return type what type does it return?

Reply:
This one is still outstanding, it returns `Markup`.
<img width="458" alt="Screenshot 2022-06-20 at 15 13 05" src="https://user-images.githubusercontent.com/53475968/174621259-d40edca3-fb68-4d2b-b97d-cf7f9ef323bc.png">